nemoguardrails/library/injection_detection/flows.v1.co

```diff
@@ -0,0 +1,23 @@
+define bot refuse to respond
+  "I'm sorry, I can't help you with that."
+
+
+define subflow reject injection
+  """
+  Reject injection attempts
+  """
+  $injection = execute reject_injection(source="output", text=$bot_message)
+
+  if $injection
+    if $config.enable_rails_exceptions
+      create event InjectionDetectionRailException(message="Injection attempt detected. Bot output contained a potential injection attempt.")
+    else
+      bot refuse to respond
+      stop
+
+
+define subflow mitigate injection
+  """
+  Omit or sanitize injection attempts from the bot.
+  """
+  $bot_message = execute mitigate_injection(source="output", text=$bot_message)
```

Comment:
source is not used in mitigate_injection, It should throw invalid argument error